nova/virt/vmwareapi/vm_util.py

```diff
@@ -704,6 +705,13 @@ def _get_device_disk_type(device):
             return constants.DEFAULT_DISK_TYPE
 
 
+def get_cluster_property(session, prop_list, cluster):
```

Comment:
Why is this cluster utility function in `vm_util` not `cluster_util`? Is it necessary to introduce a function for property-retrieval of a cluster, if you only need it once? I see other code just having that one line of code directly.

Reply:
Havent seen it Ill check